mappings/net/minecraft/network/packet/c2s/play/InventoryItemPickedC2SPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_yxgmdgda net/minecraft/network/packet/c2s/play/PickFromInventoryC2SPacket
+CLASS net/minecraft/unmapped/C_yxgmdgda net/minecraft/network/packet/c2s/play/InventoryItemPickedC2SPacket
```

Comment:
Maybe `InventoryItemRemovedC2SPacket`? I'm not 100% sure what this packet does, but `pick` makes me think of the pick block function, which i know isnt in the inventory, but can be slightly confusing.

Reply:
I'll take another look into exactly how this one is used

Reply:
so --
this packet is only sent in one situation, which is when an item is picked from the inventory (specifically the inventory, not the hotbar) in survival mode, using the middle mouse button. The game finds the associated item, if it's present in the player's inventory, and gets the associated slot index. This packet sends the slot index.

With all that in mind, I think the name is fairly accurate.

Reply:
I then agree!